### ISSUE TITLE
Make it clear that the most recently active requests are shown  at ./list/all

### ISSUE THREAD

**Issue by kat** [openaustralia/righttoknow#593]
it's not clear that  the requests shown at [List All](https://www.righttoknow.org.au/list/all) are those with the most recent activity. A heading or option of 'recently active' might help.

**Comment by henare** [comment]
Yes! I've been confused by this before too :smile: I think it would be best to raise this upstream for inclusion in core.
